lib/panel/component.js

```diff
@@ -6,7 +6,15 @@ import { $range, severityNames, sortMessages, visitMessage, openExternally, getP
 import type Delegate from './delegate'
 import type { LinterMessage } from '../types'
 
-class PanelComponent extends React.Component {
+type Props = {
```

Comment:
It's generally a good idea to do enforced type objects, like
```
type Props = {|
  delegate: Delegate,
|}
```

Normally flows allows *extra* properties in a typed object, this makes sure it doesn't